src/app/components_ngrx/group-types-panel/group-types-panel.component.ts

```diff
@@ -1,21 +1,18 @@
-import { cloneDeep } from 'lodash';
 import { Observable } from 'rxjs/Observable';
-import { Subscription } from 'rxjs/Subscription';
 
-import { Component, Input, OnChanges, OnDestroy, OnInit } from '@angular/core';
-import { ActivatedRoute, Params, Router } from '@angular/router';
```

Comment:
We are using Router in this component. Please add it back.

Reply:
Added Router in https://github.com/fabric8-ui/fabric8-planner/pull/2710/commits/cc0a547d05bed1ca96c22ba02816f66d677062df